### PR TITLE
data: HiLakshya

### PR DESCRIPTION
**Pull request by HiLakshya**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7166"></a>

### CODE REVIEW

**Review by iamthecloverly — APPROVED** [review]
LGMT :)

**Comment by HiLakshya** [comment]
>  :)

File: data/HiLakshya.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
+{
+    "name": "Lakshya Kandpal",
+    "type": "personal",
```
Comment (SaraJaoude) [inline data/HiLakshya.json:3]: this is no longer needed:
```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by HiLakshya** [comment]
Thank you for merging the PR! 
:)

**Pull request merged**